README.md

```diff
@@ -127,6 +127,14 @@ addresses* option to restrict the addresses which can be used (e.g. to limit to
 Replaces some of the record status icons so they can be more easily distinguished by people with
 colour vision deficiency.
 
+### Remember the selection of 'all status types' on the record status dashboard
+If a user selects the 'all status types' option on the record status dashboard, their selection will
+be remembered and auto-selected when they subsequently view the dashboard. This will apply per-user
+and across all projects.
```

Comment:
I am not sure what is best (can't decide myself) - should this be across all projects or pre project?

Reply:
For submit options, should it be for each project? 
Just check 'view usage' but there is no projects in it, same in Tweaker version 1.4.2 test and live RedCap in control centre

Reply:
I also try to download/upload notified alert setting from one project to the other with the same setting arm/event/form but it displayed alert error message for 'Enter a different email address' in 'Email to sent email - failure errors'.  It showed 'Email entered to send email-failure-errors is invalid' message but it is ctu-it@nottingham.ac.uk

Reply:
Alert uploads should now function correctly with custom alert senders.